cmd/usage/usage.go

```diff
@@ -86,9 +86,12 @@ var getCmd = &cobra.Command{
 			outputFile = fmt.Sprintf(outputFileFormat, startDateComponents, endDateComponents)
 		}
 
+		outputFormat = strings.ToLower(outputFormat)
+		combinedFilename := outputFile + "." + outputFormat
```

Comment:
The right thing to do here would be to check if the outputFile ends with `"." + outputFormat` (in any casing). If it does, don't do anything, use the path as is. If it does not, add the extension to the filepath.

For that, I would create a method (e.g.: `getNormalizedFilePathForFormat(filePath string, outputFormat string) string`) and write a unit test for it.

Reply:
Done

Reply:
Done